README.md

```diff
@@ -61,6 +61,34 @@ class TestCustomerModel(TestCase):
 
 Check out [documentation](<http://model-bakery.readthedocs.org/>) for more complete examples.
 
+## Contributing
```

Comment:
I am actually wondering if we should use GitHub functionality for this kind of docs:

https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file

Then it'll appear in the popups during PR creation etc..

Reply:
Fixed by cec45e8